### PR TITLE
jfc-map, set / it-academy.by / JC1-110-22 / Maksim Bondar / tasks 'Words monitoring', 'TaxService' complete

### PR DESCRIPTION
**Pull request by shiv-qa-auto**
## Pull request state

Actual status for PR:
- [x] implemented
- [ ] reviewed
- [ ] fixed
- [ ] approved

### CODE REVIEW

File: src/main/java/com/rakovets/course/java/core/practice/jcf_map/TextMonitoring.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
+package com.rakovets.course.java.core.practice.jcf_map;
+
+import java.util.*;
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/java/core/practice/jcf_map/TextMonitoring.java:3]: Don't use * import

File: src/test/java/com/rakovets/course/java/core/practice/jcf_map/TextMonitoringDemo.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package com.rakovets.course.java.core.practice.jcf_map;
+
+public class TextMonitoringDemo {
```
Comment (rakovets) [inline src/test/java/com/rakovets/course/java/core/practice/jcf_map/TextMonitoringDemo.java:3]: Best way: tests

**Review by rakovets — CHANGES_REQUESTED** [review]
Need to add theme 'Set'

File: src/main/java/com/rakovets/course/java/core/practice/jcf_map/TextMonitoring.java (hunk 1 above)
Comment (shiv-qa-auto) [reply src/main/java/com/rakovets/course/java/core/practice/jcf_map/TextMonitoring.java:3]: Fixed

**Pull request closed without merging**